technologies/job/trifacta/login-auth/instanceActions.js

```diff
@@ -0,0 +1,280 @@
+const axios = require('axios');
+const https = require('https');
+const fs = require('fs');
+const extract = require('extract-zip');
+const rimraf = require('rimraf');
+const { Response, JobStatus, Log } = require('@saagie/sdk');
+
+const agent = new https.Agent({  
+  rejectUnauthorized: false
+});
+
+const getExistingOutputObjectForJob = async (job) => {
+  try {
+    const { data: result } = await axios.get(
+      `${job.featuresValues.endpoint.url}/v4/outputObjects`,
+      {
+        httpsAgent: job.featuresValues.endpoint.ignoreSslIssues && job.featuresValues.endpoint.ignoreSslIssues.id ? agent : null,
```

Comment:
This line is duplicated 12 times, I think you can probably have an util function somewhere to avoid these duplicates 🙂